apps/remote_control/lib/lexical/remote_control/build/error_support.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Lexical.RemoteControl.Build.ErrorSupport do
+  def context_to_position(context) do
```

Comment:
Is this coming from metadata? Context is such an unhelpful word

Reply:
it is from stack_context or [string_to_quoted](https://github.com/elixir-lang/elixir/blob/927b10df80ee1c1c7396e68efe00d06bc3e80420/lib/elixir/lib/code.ex#L1147) 's `meta`